tv_apps/smart-system/js/preview_window.js

```diff
@@ -100,8 +99,18 @@
       return;
     }
     if ((evt.keyCode === 27 || evt.key === 'Escape') &&
-        !evt.embeddedCancelled) {
-      this.kill();
+        !evt.embeddedCancelled && !this.config.url.startsWith('app://')) {
```

Comment:
You should kill the app directly if `this.config.url.startsWith('app://')` is true.